addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/CommandInjectionScanRule.java

```diff
@@ -98,7 +98,8 @@ public class CommandInjectionScanRule extends AbstractAppParamPlugin
                         CommonAlertTag.toMap(
                                 CommonAlertTag.OWASP_2021_A03_INJECTION,
                                 CommonAlertTag.OWASP_2017_A01_INJECTION,
-                                CommonAlertTag.WSTG_V42_INPV_12_COMMAND_INJ));
+                                CommonAlertTag.WSTG_V42_INPV_12_COMMAND_INJ,
+                                CommonAlertTag.TEST_TIMING));
```

Comment:
This will be included in all alerts raised by the rule which IMHO is misleading.

Reply:
Well it looked like we had decided one example alert was sufficient

Reply:
I can look at adding it differently.

It kinda needs to be on the rule for docs/site stuff (maybe I can remove it when it isn't a timing alert). Also kinda why I'd asked about rule vs alert tags way back.

I think the same applies for the shell shock rule.

Reply:
It can be done similar to this: https://github.com/zaproxy/zap-extensions/pull/3798/files

Reply:
I'll have to tweak a few others but how's that look for CMDi now?